src/mixins/itext_behavior.mixin.js

```diff
@@ -672,10 +670,6 @@
 
       this.shiftLineStyles(lineIndex, +1);
 
-      if (!this.styles[lineIndex + 1]) {
-        this.styles[lineIndex + 1] = {};
-      }
-
```

Comment:
no need to create an empty style object for a line just because maybe we will put something into it.